src/template.html

```diff
@@ -0,0 +1,46 @@
+<!DOCTYPE html>
+<html> 
+  <head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Hello world!</title>
+  </head>
+  <body>
+    <h1>Leaderboard</h1>
+
+  <section class="board">
+
+    <div class="container">
+      <div class="recent-score">
+        <h2>Recent scores</h2>
+        <button>Refresh</button>
```

Comment:
- Please, let's align the button correctly to match it position as seen from the wireframe below. kindly see the images below:

*wireframe*
![Screenshot from 2023-03-07 14-27-15](https://user-images.githubusercontent.com/59397159/223436113-40f1d4ea-cd90-4d48-b4c5-ea5c1ed50b3c.png)

*your design*
![Screenshot from 2023-03-07 14-02-13](https://user-images.githubusercontent.com/59397159/223436124-ea4db431-5e12-4638-aea1-7278a1459c2f.png)